boards/riscv/tlsr9518adk80d/doc/index.rst

```diff
@@ -81,7 +81,7 @@ The following example projects are supported:
 - samples/hello_world
 - samples/synchronization
 - samples/philosophers
-- samples/event_manager
+- samples/application_development/event_manager
```

Comment:
probably we need to have another move, this is being introduced as a subsystem, so it needs to end up in samples/subsys/event_manager. Also, I am not sure why you have it in this file

Reply:
Ok, it can be moved there.